charts/camunda-platform/charts/web-modeler/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+description: Web Modeler Helm Chart for Kubernetes
+name: web-modeler
+version: 0.1.0
```

Comment:
I was not sure about the chart version because Web Modeler Self-Managed is still in beta status and does not follow the Camunda 8 versioning scheme yet. So I went with `0.1.0`.

Reply:
These sub chart values doesn't matter since we only package one chart for camunda platform. We normally align these values with the parent.

Reply:
Thanks, I changed the version to `8.1.2`.